app/controllers/checkout_controller.rb

```diff
@@ -62,6 +68,22 @@ def update
 
   private
 
+  # Check if items present in cart have been removed from the OC
+  # before checkout has been completed
+  def recheck_stock_availability
+    outgoing_variants = @order.order_cycle.exchanges.outgoing.first.variants.pluck(:id)
+    items_available = true
+
+    @order.line_items.each do |i|
```

Comment:
weird that rubocop doesn't complaint about `|i|`. I'd rather name it `line_item` because as it is, it makes you think it is an index and this is not the case.